python/ray/util/sgd/torch/training_operator.py

```diff
@@ -142,6 +177,14 @@ def train_epoch(self, ...):
             batch_info.update(info)
             metrics = self.train_batch(batch, batch_info=batch_info)
 
```

Comment:
Ideally, here we just have `self.process_batch(metrics)`. This makes it easier for someone to extend the operator while not losing the functionality of the package.